tsconfig.json

```diff
@@ -9,10 +9,8 @@
     "module": "es2015",
     "moduleResolution": "node",
     "noUnusedLocals": true,
-    "outDir": "dist",
     "plugins": [{ "name": "typescript-styled-plugin" }],
     "rootDir": "src",
-    "sourceMap": true,
```

Comment:
Is this not needed to generate source-maps?

Reply:
Actually, let me see. For some reason I had presumed that Babel would handle this via the preset but let me double check.

Reply:
@alloy - looks good: 

![screen shot 2018-02-22 at 2 32 04 pm](https://user-images.githubusercontent.com/236943/36567959-3605b238-17dd-11e8-8dd2-128d855833bc.png)

Reply:
👍